vlib/v/checker/tests/function_arg_mutable_err.out

```diff
@@ -1,4 +1,4 @@
-vlib/v/checker/tests/function_arg_mutable_err.vv:1:18: error: mutable arguments are only allowed for arrays, maps, structs and pointers
+vlib/v/checker/tests/function_arg_mutable_err.vv:1:18: error: mutable arguments are only allowed for arrays, maps, aggregates and pointers
```

Comment:
V doesn't define the term aggregate anywhere, better to be explicit (structs, interfaces)

Reply:
Now done